src/Control/Email/Email.php

```diff
@@ -197,13 +197,14 @@ public function __construct(
         $this->data = ViewableData::create();
     }
 
-    private function getDefaultFrom(): string
+    private function getDefaultFrom(): string|array
     {
         // admin_email can have a string or an array config
         // https://docs.silverstripe.org/en/4/developer_guides/email/#administrator-emails
         $adminEmail = $this->config()->get('admin_email');
         if (is_array($adminEmail) && count($adminEmail ?? []) > 0) {
-            $defaultFrom = array_keys($adminEmail)[0];
+            $email = array_keys($adminEmail)[0];
+            $defaultFrom = [$email => $adminEmail[$email]];
```

Comment:
Should we validate here that `array_keys($adminEmail)[0]` is in fact a string (or even go so far as to validate it's a valid RFC email address?

I know the second part of that goes beyond the original purpose of this PR so I'm okay if we say we want to do that but make a card to handle it later - but the former part of that seems like it falls in line with this PR's purpose if we want to do it.

Reply:
Validation happens within symfony so we don't need to do this here, nor should really should we since we're leveraging symfony to do the heavy lifting

e.g. the following config
```
SilverStripe\Control\Email\Email:
  admin_email:
    aa: 'Admin-email'
```

and the following script
```
$email = new Email();
$email->to('to@example.com')->subject('test')->setBody('abc')->send();
```
Will result in 
`[Emergency] Uncaught Symfony\Component\Mime\Exception\RfcComplianceException: Email "aa" does not comply with addr-spec of RFC 2822.`

Reply:
What about

```yml
SilverStripe\Control\Email\Email:
  admin_email:
    - 'me@example.com'
```